oauthlib/oauth2/rfc6749/grant_types/authorization_code.py

```diff
@@ -135,7 +139,7 @@ def create_authorization_response(self, request, token_handler):
         HTTP redirection response, or by other means available to it via the
         user-agent.
 
-        :param request: oauthlib.commong.Request
+        :param request: oauthlib.common.Request
```

Comment:
`:param` should be used for the description.
For specifying the parameter type if a description is missing, use `:type`.
See example at: https://pythonhosted.org/an_example_pypi_project/sphinx.html#function-definitions

Reply:
in my review I saw the majority of `request` defined as 

    :param request: oauthlib.common.request

i'd be happy to standardize these to a `:param` + `:type`, however I do not believe that mixture has been used in oauthlib before, and I don't know what the text for `:param` should be.

Reply:
Hi @jvanasco, sorry for the later response. Could you standardize those please ? Would be very great for the project's health :-) 

I suggest to use as below:

    :param request: HTTP request
    :type: oauthlib.common.Request

Reply:
ok. since we're discussing this though... imho, this is a little unclear for people unfamiliar with the library, as it uses "request" to define `request` - and people coming from different web frameworks will be reading this in the context of their own "request". it is not clear the oauthlib `request` is a secondary object built off the `request` object/data in their framework.

    :param request: HTTP request

what if it were something like:

    :param request: the active HTTP request object
    :param request: parsed current HTTP request object
    :param request: parsed HTTP request
    :param request: a HTTP request parsed for oAuth data
    :param request: an object of active oAuth data parsed from a HTTP request

I'm sure you could come up with something better. This has actually been a bit of a pet peeve for me in the docs though - so I'd actually love to make this as clear as possible.

Reply:
I like the `OAuthlib request`. I can replace things globally with that.